liquibase-service/src/main/resources/db/rdb_modern/views/007-v_code_value_general-001.sql

```diff
@@ -0,0 +1,13 @@
+use rdb_modern;
```

Comment:
Similar- removal of rdb_modern reference

Reply:
@upasanapattnaik-eq not needed. See comments above